### PR TITLE
React: Cherry-Pick Commit #1

### PR DESCRIPTION
**Pull request by jnaviask**
## Link to Issue
Does not fix but affects #3225.

## Description of Changes
- Cherry-picks many commits from master onto react. Mostly the easy ones.

## Test Plan
- Verified key changes (proposals) via clicking around. Most changes not UI-related, but should ensure all UI changes are valid.

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/proposals/voting_results.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import m from 'mithril';
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/proposals/voting_results.tsx:4]: remove

Comment (jnaviask) [reply packages/commonwealth/client/scripts/views/components/proposals/voting_results.tsx]: fixed

File: packages/commonwealth/client/scripts/views/components/sidebar/governance_section.tsx [hunk 2]
```diff
@@ -88,10 +87,22 @@ export const GovernanceSection = () => {
...
+    const showProposals =
+      (isNotOffchain &&
+        app.chain?.base === ChainBase.Substrate &&
+        app.chain.network !== ChainNetwork.Darwinia) ||
+      (app.chain?.base === ChainBase.CosmosSDK &&
+        app.chain.network !== ChainNetwork.Terra) ||
+      app.chain?.network === ChainNetwork.Sputnik ||
+      app.chain?.network === ChainNetwork.Compound ||
+      app.chain?.network === ChainNetwork.Aave;
+>>>>>>> 084c049fe (Remove chain-event Substrate code for related deprecated client code (#3043))
```
[the remark this answers is not in the document]
Comment (jnaviask) [reply packages/commonwealth/client/scripts/views/components/sidebar/governance_section.tsx]: fixed

**Review by CowMuon — APPROVED** [review]

**Comment by CowMuon** [comment]
WTG Jake, this is a really good start, and especially helpful to have the list of [remaining commits on master not on react](https://github.com/hicommonwealth/commonwealth/issues/3225#issuecomment-1481507590), a number of which are @alexyoung23j 's, so the expectation here is he should be able to have most if not all of the remaining commits merged by EOW (ie tomorrow.)

**Comment by jnaviask** [comment]
To avoid rewriting history = removing all the above commits, I am going to do a standard rather than a squash merge into master via the command line, then close this PR.

**Pull request closed without merging**